PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/README.md

```diff
@@ -259,4 +259,17 @@ integration, telemetry connectivity, and Azure Toolkit integration.
   that require it. Otherwise, it is not necessary to authenticate non-Azure Open-AI clients.
   Please refer to
   the [KeyCredential Class documentation](https://learn.microsoft.com/java/api/com.azure.core.credential.keycredential?view=azure-java-stable)
-  for more information.
\ No newline at end of file
+  for more information.
+
+14. #### Use sync client operation if calling block methods on an async client.
+
+- **Anti-Pattern**: Using `block()` on an async client. This practice turns an asynchronous operation into a synchronous
```

Comment:
Instead of `block()` we should replace it with "blocking calls" as there are other methods like `blockFirst()`, `blockLast()` etc. that are also blocking.